analyses/infercnv-consensus-cell-type/references/normal-references/reference-cell-groups.tsv

```diff
@@ -0,0 +1,53 @@
+consensus_annotation	reference_cell_group
+adipocyte	adipocyte
+B cell	immune
+lymphocyte of B lineage	immune
+mature B cell	immune
+naive B cell	immune
+memory B cell	immune
+chondrocyte	stromal
+dendritic cell	immune
+endothelial cell	endothelial
+blood vessel endothelial cell	endothelial
+microvascular endothelial cell	endothelial
+epithelial cell	epithelial
+ecto-epithelial cell	epithelial
+keratinocyte	epithelial
+squamous epithelial cell	epithelial
+epidermal cell	epithelial
+fibroblast	stromal
+hematopoietic precursor cell	HSPC

```

Comment:
I'm torn on if we should combine this with the other stem cell group, but I think it's probably fine since realistically anything that's a stem cell could be a cancer cell so I would never use those in a normal reference.

Reply:
Yeah, that's a good point. While they are super closely related, for reference building in this context, we'd want to keep them separate.